### PR TITLE
Asset Compilation Using Webpack

### PR DESCRIPTION
**Pull request by pet1330**
This PR is just to organise the frontend, the backend has not been changed.

- Removed the static directory
- Pulling in up to date libraries through npm
- Bundled and minified & uglify production assets (reducing https request)
- restructuring development assets into a resources directory
- removing all js from the index.html file

### CODE REVIEW

File: fd_webserver/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "fd_webserver",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": {
```
Comment (marc-hanheide) [inline fd_webserver/package.json]: can there only be one?

**Comment by pet1330** [comment]
@marc-hanheide Can you review this?

**Review by marc-hanheide — COMMENTED** [review]
the python 3.x stuff is fine. How about my comment on the authors @pet1330 ?

File: fd_webserver/package.json (hunk 1 above)
Comment (pet1330) [reply fd_webserver/package.json]: Yes, only one author, everyone else should be listed as `contributors`. I was meant to make you the author and set Ashley and myself as contributors, this was just my pre-filled snippet. I'll change it now :smiley:

**Pull request merged**